### PR TITLE
#167484577 Install and setup dotenv

### PR DESCRIPTION
**Pull request by raymond42**
### What does this PR do?
- Install and setup dotenv for the API to manage environment variables
### Description of Task to be completed?
- installed dotenv package
- configured environment variables
### How should this be manually tested?
- Clone the repo, run npm install , create .env file in the root folder then save from there your variables
### any background context you want to provide?
- N/A

### What are the relevant pivotal tracker stories?
#167484577

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -33,4 +33,6 @@ node_modules
 
 # Optional REPL history
 .node_repl_history
-
+ 
+#  dotenv
```
Comment (mkiterian) [inline .gitignore:37]: delete this commented out line

**Review by mkiterian — CHANGES_REQUESTED** [review]

**Comment by harerakalex** [comment]
Looks good, so implement feedback from @mkiterian

**Comment by Jkadhuwa** [comment]
All looks good. Just work on the TTL's requested change.

**Review by Jkadhuwa — COMMENTED** [review]
All looks good. Just work on TTL's requested change

File: .gitignore (hunk 1 above)
Comment (raymond42) [reply .gitignore]: done

**Comment by raymond42** [comment]
> Looks good, so implement feedback from @mkiterian

done

**Comment by raymond42** [comment]
> Looks good, so implement feedback from @mkiterian

done

File: .gitignore (hunk 1 above)
Comment (raymond42) [reply .gitignore:37]: > delete this commented out line

done

**Review by mkiterian — APPROVED** [review]
Looks good

**Pull request merged**